t/lib/server.lua

```diff
@@ -73,6 +73,7 @@ end
 function _M.plugin_proxy_rewrite()
     ngx.say("uri: ", ngx.var.uri)
     ngx.say("host: ", ngx.var.host)
+    ngx.say("http_host: ", ngx.var.http_host)
```

Comment:
It is not necessary to add this line of code. Try this `uri`: https://github.com/apache/apisix/blob/master/t/lib/server.lua#L111